README.md

```diff
@@ -1,18 +1,28 @@
 # GOG Galaxy Minecraft Integration
 
-An Minecraft integration for GOG Galaxy 2.0.
+An Minecraft integration for GOG Galaxy 2.0. Installable via GOG Galaxy (see GIF below).
 
-![example](example.gif)
-
-## Build
+[![1.0.1](https://img.shields.io/badge/version-1.0.1-blue)](https://GitHub.com/urwrstkn8mare/galaxy-riot-integration/releases/)
+[![MIT License](https://img.shields.io/github/license/TouwaStar/Galaxy_Plugin_Minecraft)](https://github.com/urwrstkn8mare/TouwaStar/Galaxy_Plugin_Minecraft/fog_release/LICENSE)
+[![1.0.1 Downloads](https://img.shields.io/github/downloads/FriendsOfGalaxy/galaxy-integration-minecraft/1.0.1/total.svg)](https://github.com/FriendsOfGalaxy/galaxy-integration-minecraft/releases)
 
-`python3 build.py -r` This will make a zip with the integration and all necessary packages for the operating system run on. You can specify a platform but I've found that to not work well when building for another platform on a platform (eg. building for mac on windows). For more information run `python build.py -h`.
+![example](example.gif)
 
 ## FAQ
 
 _How to change what games I own (owned games selection)?_ Just disconnect and reconnect. The play time should be kept. If there is an issue please submit an issue.
 
+## Todo
+
+- [ ] Finish [INSTALL_MULTIMC.md](INSTALL_MULTIMC.md)
```

Comment:
What should be finished here? Maybe prerequisites section with link to Java installation? Linking MultiMC Wiki page is good idea (not the best UX, but robust to any changes in external source)

Reply:
sth like:


```markdown
## Prerequisites:
MultiMC requires Java to be installed. Get installer here:
<java link I proposed last time>

## MultiMC Installation wiki:
<wiki link>

```

Reply:
My original plan was to link to the MultiMC Wiki page but also have some additional instructions detailing prerequisites (eg. Java), additional help on how to get the path to MultiMC to put into GOG Galaxy, and more. So far I haven't had time to do it but I've created the file for future development. I realise it may be better to just take the user directly to the [Getting Started](https://github.com/MultiMC/MultiMC5/wiki/Getting-Started) page until the [`INSTALL_MULTIMC.md`](https://github.com/TouwaStar/Galaxy_Plugin_Minecraft/pull/10/files#diff-869195911ccf6d7d5891f63309953cd7) is ready.